packages/cli/src/cmds/preview/depsChecker/cliTelemetry.ts

```diff
@@ -32,19 +39,28 @@ export class CLITelemetry implements IDepsTelemetry {
   }
 
   public sendUserErrorEvent(eventName: DepsCheckerEvent, errorMessage: string): void {
-    // TODO: send user error event
+    const error = new UserError(eventName, errorMessage, this._telemetryComponentType);
+    cliTelemetryInstance.sendTelemetryErrorEvent(eventName, error, this.addCommonProps());
   }
 
   public sendSystemErrorEvent(
     eventName: DepsCheckerEvent,
     errorMessage: string,
     errorStack: string
   ): void {
-    // TODO: send system error event
+    const error = new SystemError(
+      eventName,
+      `errorMsg=${errorMessage},errorStack=${errorStack}`,
```

Comment:
why do you want to put callstack into error message?

Reply:
1. This code are from the extension. 
2. The reason in the extension is that the error stack is not in the telemetry properties. We put the stack in the error message to work around this issue.

Does CLI telemetry have this issue too? If not, we can remove this.